hooks/playbooks/compute-iscsi-config.yml

```diff
@@ -0,0 +1,19 @@
+---
+- name: Change iscsi login timeout on compute nodes
+  hosts: computes
+  gather_facts: false
+  tasks:
+    - name: Change timeout in iscsid.conf
+      become: true
+      ansible.builtin.lineinfile:
+        path: /etc/iscsi/iscsid.conf
+        line: "{{ item }}"
+      loop:
+        - 'node.session.initial_login_retry_max = 3'
```

Comment:
Couldn't we add a couple of Ansible variables with these values as defaults?
Allowing adding extra lines passing a list to the playbook sounds good too :)

Reply:
I'm OK with that, I just didn't want to over engineer things.
I have already agreed with @tosky to make a follow up PR to make some improvements here, but we wanted to merge this which we have already tested first to unblock CI jobs.
What do you say I make those changes as a follow up?

Reply:
I'm fine with that approach if someone is blocked.